src/evaluators/ConditionalExpression.js

```diff
@@ -48,28 +48,20 @@ export default function(
       return realm.evaluateNodeForEffects(consequent, strictCode, env);
     },
     () => {
-      let completion = env.evaluateCompletion(alternate, strictCode);
+      let completion = env.evaluateCompletionDeref(alternate, strictCode);
```

Comment:
If this used evaluate rather than evaluateCompletionDeref, you could dispense with the check for AbruptCompletion, but you'd need to bring back the check for Reference. It sounds like we need a helper called evaluateDeref

Reply:
That sounds good.